pom.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>se.lfv.reqstool</groupId>
+    <artifactId>reqstool-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <version>0.0.3</version>
+
+    <name>${project.artifactId}</name>
+    <description>Reqstool - Maven Plugin</description>
+    <url>https://github.com/Luftfartsverket/reqstool-java-maven-plugin</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://www.opensource.org/licenses/mit-license</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>LFV System Development Team</name>
```

Comment:
What name do we use in the other pom.xml and pyproject.toml?

Either way, it's not Development but Developer Team.